client/src/redux/actions/credentialsActions.js

```diff
@@ -1,40 +1,40 @@
-import * as types from '../constants/credentialsConstants';
-import credentialsApi from '../../services/credentialsApi';
+import { credentialsTypes as types } from '../constants';
```

Comment:
I'd rather not change this from credentialsTypes to types here. More readable below if we leave it as credentialsTypes, imo.

Reply:
We can leave, sure, I can do a replace on those. Probably should make sure to change them across the board. Still have all of the commits so cannn do!